web_src/js/utils/dom.ts

```diff
@@ -297,7 +297,7 @@ export function createElementFromAttrs(tagName, attrs) {
   const el = document.createElement(tagName);
   for (const [key, value] of Object.entries(attrs)) {
     if (value === undefined || value === null) continue;
-    if (value === true) {
+    if (value === true || value === false) {
```

Comment:
```suggestion
    if (typeof value === "boolean") {
```

Reply:
![image](https://github.com/user-attachments/assets/4866ec48-f924-489e-a6e0-b135e934177c)

Reply:
![image](https://github.com/user-attachments/assets/30926e9e-f5c9-437e-a5cb-6cf3b151d51b)

Reply:
![image](https://github.com/user-attachments/assets/91bdde2a-0d2f-4d43-9719-2f56c0a521da)

Reply:
![image](https://github.com/user-attachments/assets/c41ca48b-bcf1-4c9e-baff-bf3c94e0aeaf)

Reply:
![image](https://github.com/user-attachments/assets/97b6e8c7-ca16-42c2-9a94-9efb7a3c8b9c)

Reply:
ps: Using `===` for `data.find(x => typeof x === "boolean")` doesn't make difference.

Reply:
```suggestion
    if (typeof value === 'boolean') {
```

Reply:
By testing with more cases and browsers, it seems that there is no always-winner. I guess it is related to that the browser's JIT generates different machine code for different logic.

But I still dislike the `typeof` here, it introduces a string literal `'string'` which is unnecessary. `=== true ||  === false` is clear enough.

If you'd really like to change it, the string needs to be quoted by a single quote (done in 82510da3ef85232fc88241af3f0e5c53ad986803)